### DISCUSSION TITLE
GUI Improvments :D

### DISCUSSION THREAD

**Discussion by STPv22** [eaglerforge/EaglerForgeInjector#35] [Ideas]
![image](https://github.com/user-attachments/assets/d6335bd7-3bad-4c81-80bd-f3cdcd83a9b9)
I'm done with the styling, just have to update js

**Comment by STPv22** [comment]
![image](https://github.com/user-attachments/assets/8e5a2942-ed31-4463-b8ce-adfd9d9fe812)
there's a couple things I want to add still but I think it looks pretty good so far @ZXMushroom63

**Reply by STPv22** [reply]
also sorry its not on discord

**Reply by OeildeLynx31** [reply]
I'll make a second design based on this one, this is a good start

**Reply by STPv22** [reply]
alr, noice

**Reply by STPv22** [reply]
opened a pull request

**Comment by mmercer613-sudo** [comment]
hey you really should put more items and blocks into eaglercraft like the blocks from 1.21 deep dark update and netherite armor and tools and just what i was thinking maybe change the glass textures to make them look like the ones from the real game

**Comment by STPv22** [comment]
Just use tuffclient, they have sume good backwards compatibility stuff
